### PR TITLE
[ADD]:sale_request: add the same validation that exist in sale order l… task#2420

### PR DESCRIPTION
**Pull request by jesus01x**
…ine model, they validate if you have enought stock in a specific warehouse

### CODE REVIEW

File: sale_request/models/sale_request.py [hunk 1]
```diff
@@ -269,13 +270,121 @@ def name_get(self):
...
+        if self.product_id.type == 'product':
+            precision = self.env['decimal.precision'].precision_get(
+                'Product Unit of Measure')
+            product = self.product_id.with_context(
+                warehouse=self.request_id.warehouse_id.id,
+                lang=self.request_id.partner_id.lang or
+                self.env.user.lang or 'en_US'
+            )
+            product_qty = self.product_qty
+            # product_qty = self.product_uom._compute_quantity(self.product_uom_qty, self.product_id.uom_id)
```
Comment (alan196) [inline sale_request/models/sale_request.py]: why you comment this?

File: sale_request/i18n/es.po [hunk 2]
```diff
@@ -370,6 +379,12 @@ msgstr "Vincular pedido"
 msgid "Main Attachment"
 msgstr "Adjuntos principales"
 
+#. module: sale_request
+#: code:addons/sale_request/models/sale_request.py:369
+#, python-format
+msgid "Make To Order"
+msgstr ""
```
Comment (alan196) [inline sale_request/i18n/es.po]: Pleas add all translations

**Review by alan196 — CHANGES_REQUESTED** [review]

File: sale_request/models/sale_request.py (hunk 1 above)
Comment (jesus01x) [reply sale_request/models/sale_request.py]: I forget remove that line, it already

Comment (alan196) [reply sale_request/models/sale_request.py]: Don't remove this line, it's important to verify the quantity in the product default UoM

**Pull request merged**